osgar/drivers/tankmaria.py

```diff
@@ -0,0 +1,167 @@
+"""
+  Driver for a small tank Maria
+
+  Text Communication Protocol
+  Version 0 uses ASCII protocol between the host computer (JetsonNano) and Arduino motor controller.
+  There is no speed control implemented on Arduino at the moment, so raw PWM is used:
+     "<PWM left> <PWM right> <desired LED status>\n"
+  The value for PWM can in in -255..255.
```

Comment:
`can in in` typo?

Reply:
The text ` The value for PWM can in -255..255.` still does not look like a correct sentence.